plugins/testrail-integration-plugin/content.yaml

```diff
@@ -0,0 +1,260 @@
+title: TestRail Integration Plugin
+author: railflow
+tags:
+  - publish
+  - testrail
+  - reporting
+  - railflow
+  - test
+  - junit
+  - cucumber
+  - testng
+logo: testrail.svg
+repo: https://railflow.io
```

Comment:
Thanks for the review. Unfortunately, I can't since this is a commercial product with closed sources

Reply:
Hi @sergey-oplavin I'm afraid we can only accept open source plugins in our index - sorry

generally in this scenario what is done is the drone plugin code is open source but the plugin might make use of closed source CLI commands or API requests

hopefully this can be worked around  on your end :)

Reply:
Thanks for the explanation @d1wilko , let me get back to the team and see what we can do to fulfill the requirements!

Reply:
@d1wilko just one clarification - our Docker image is based on node image and we just install https://www.npmjs.com/package/railflow package and provide entrypoint script for it.  
If we make our Docker image repo open - will you be able to accept it?